src/python/WMCore/WMSpec/StdSpecs/StepChain.py

```diff
@@ -445,6 +462,11 @@ def validateSchema(self, schema):
                                                couchDBName=schema["CouchDBName"],
                                                getOutputModules=True)
 
+        if 'KeepOutput' in schema[stepName] and strToBool(schema[stepName]['KeepOutput']) is False:
+            msg = "Dropping the output of the last step is prohibited.\n"
+            msg += "Set the 'KeepOutput' value to True and try again."
+            self.raiseValidationException(msg=msg)
```

Comment:
Did not look at the base class, but do we really need a function to raise an exception?